web/src/rest.ts

```diff
@@ -37,11 +40,7 @@ const dandiRest = new Vue({
     };
   },
   computed: {
-    schemaVersion(): string {
-      // Use injected $store instead of importing to
-      // avoid dependency cycle
-      return this.$store?.direct.getters.dandiset.schemaVersion;
-    },
+    ...mapState(useDandisetStore, ['schemaVersion']),
```

Comment:
We should probably migrate this component to use composition API as well, but that's out of scope for this PR. Do you mind filing an issue?

Reply:
https://github.com/dandi/dandi-archive/issues/1267